plutus-streaming/README.md

```diff
@@ -0,0 +1,29 @@
+# Plutus Streaming PoC
```

Comment:
You need to cherrypick https://github.com/input-output-hk/plutus-apps/commit/c9a67bb47f8760019ed6337eded733e882616c15 to fix the readme.